draft-ietf-scitt-architecture.md

```diff
@@ -919,13 +919,12 @@ All contents exchanged between actors is protected using secure authenticated ch
 
 #### Signed Statements and Their Registration
 
-The Transparency Service is trusted with the confidentiality of the Signed Statements presented for registration.
-Some Transparency Services may publish every Transparent Statement in their logs, to facilitate their dissemination and auditing.
-Others may just return Receipts to clients that present Singed Statements for registration, and disclose the ledger only to auditors trusted with the confidentiality of its contents.
+The Transparency Service is trusted with the confidentiality of the Signed Statements presented for registration. The act of registration will always record in the
+logs the Transparent Statement and associated resolution evidence.  There is a use case where the logs are completely publicly accessible, and another where the audit requires elevated permission (this could be for privacy or for mitigation of servicing expensive operations). In either case the returned Receipts to the client will both facilitate simple evaluation of meeting the identity policy bar and act as a key to allow query during the elevated deep audit.
```

Comment:
Are we sure that the transparent statement goes in the log?

That seems a bit meta, since the signed statement gets a receipt when it goes in the log, and then the signed statement with the embedded receipt becomes a transparent statement, if that goes in the log, the receipt you would get would be for a transparent statement, not a signed statement.

My mental model is:

signed statement 
-> registration 
-> (log entry, with evidence) 
-> receipt 
-> (notary signature, from transparency service)
-> transparent statement

> "The act of registration will always record in the logs the Transparent Statement and associated resolution evidence."

I think this should be:

"Successful registration implies that the **Signed Statement** and any associated evidence, have been appended to the transparency log"

Reply:
Re: "mental model" sequence, I see the production of a receipt as a two-step process: (1) the transparency service signs and persists the log entry; (2) anyone (e.g. the transparency service) can extract from the extended log the receipt by collecting the signature and intermediate hashes.

Reply:
I like your description and edits to the text. The mental model "registration" will likely need to contact a user authorization service that will effectively authorize the user that has an established session with the registry, based on a second connection to a separate service that will verify that the user has the authorization to submit the statements on behalf of the entity. Perhaps this is not relevant to this PR though.